searx/static/themes/simple/src/js/main/keyboard.js

```diff
@@ -43,6 +43,29 @@ searxng.ready(function () {
     }
   });
 
+  searxng.on('.result', 'keydown', function (e) {
```

Comment:
Why not use `.image-detail-open` class directly then?
Plus with that the next 2 lines can be removed (no check for image result needed)

Reply:
that all works same, because `.image-detail-open` contains `.result`.

Reply:
However, it seems has a new problem:

It's about the focus point: 
If user focus on the image (e.g., click the image), we won't catch the event when binding on `'.result'`
Otherwise, if we bind it on `.detail`(which is the pannel that display the image) but the user didn't do anything, then the focus is still on `.result`, we can't catch it on `.detail`.

Reply:
maybe we can bind it on `body` and use your `hasImageInDetail` to check whether a image result is opened.

```js
  function hasImageInDetail () {
    return document.querySelector('#results.image-detail-open') !== null;
  }
```
what do you think?

Reply:
> It's about the focus point: If user focus on the image (e.g., click the image), we won't catch the event when binding on `'.result'` Otherwise, if we bind it on `.detail`(which is the pannel that display the image) but the user didn't do anything, then the focus is still on `.result`, we can't catch it on `.detail`.

Not sure what you mean exactly, but in my tests locally just binding on `.image-detail-open` seems to work properly.
But yes the user needs to click on one image first (resulting in showing `detail` and adding `.image-detail-open`), that's what's expected, isn't it?

Reply:
for the first comment, i agree with you.
for the second, i think i should make a video to explain this:

https://github.com/searxng/searxng/assets/88757735/efebe378-fde2-4547-8d43-e3fbc48a05ec

You see, that's the problem and we should solve it.

Reply:
What are the steps? On my end that seems to work as intended, I click on an image and then use the arrow keys. Maybe I missed something?
I don't have any condition on `e.target` locally though, as said in the reviews

Reply:
well, it's really hard to explain, so i think we should put it away right now. 
We can solve it later ?

Reply:
> What are the steps? On my end that seems to work as intended, I click on an image and then use the arrow keys. Maybe I missed something? I don't have any condition on `e.target` locally though, as said in the reviews

maybe you missed the step of clicking the blank when the image preview is opened.

Reply:
i think we can start a new issue or discussion to talk about this.